src/main/kotlin/com/atlassian/performance/tools/report/api/judge/ActionReportImpl.kt

```diff
@@ -0,0 +1,18 @@
+package com.atlassian.performance.tools.report.api.judge
+
+import com.atlassian.performance.tools.jiraactions.api.ActionType
+import com.atlassian.performance.tools.report.api.junit.JUnitReport
+import java.nio.file.Path
+
+internal class ActionReportImpl(
```

Comment:
Sorry, but "Impl" is literally the worst implementation name. It shows no direction on how to extend it. It does not differentiate from other implementations. If we cannot find a better name then it is a strong indication that the interface should not exist.

Reply:
BuiltInActionReport ?

Reply:
I will try to come up with a better name